Cargo.toml

```diff
@@ -5,23 +5,24 @@ authors = ["CPerezz <carlos@dusk.network>", "Victor Lopez <victor@dusk.network>"
 edition = "2018"
 
 [dependencies]
-dusk-pki = {git = "https://github.com/dusk-network/dusk-pki", tag = "v0.4.1", default-features = false}
-poseidon252 = {git = "https://github.com/dusk-network/poseidon252", tag = "v0.15.0", default-features = false}
-dusk-bls12_381 = {version = "0.3", default-features = false}
-dusk-jubjub = {version = "0.5", default-features = false}
-dusk-plonk = {version = "0.3", features = ["trace-print"], optional = true}
+dusk-pki = {git = "https://github.com/dusk-network/dusk-pki", tag = "v0.5.1", default-features = false}
+poseidon252 = {git = "https://github.com/dusk-network/poseidon252", tag = "v0.17.0", default-features = false}
+dusk-bls12_381 = {version = "0.6", default-features = false}
+dusk-jubjub = {version = "0.8", default-features = false}
+dusk-plonk = {version = "0.5", features = ["trace-print"], optional = true}
 num-bigint = {version = "0.3", optional = true } 
 num-traits = {version = "0.2", optional = true } 
-plonk_gadgets = {git = "https://github.com/dusk-network/plonk_gadgets", tag = "v0.4.3", optional = true}
+plonk_gadgets = {git = "https://github.com/dusk-network/plonk_gadgets", tag = "v0.5.0", optional = true}
 rand_core = {version = "0.5", default-features = false}  
-lazy_static = "1.4.0"
+lazy_static = "1"
```

Comment:
I'm not totally convinced we should only mark major and have minor for external libs always updated. I think we should stick with `major.minor`.

Any argument cons that, and pro to use only the major?

(The same for anyhow, it's a general thought)

Reply:
Indeed anyhow recommends to use just major in `Cargo.toml`.

The point is simple. Once we're in 1.X minor releases will not break API and therefore, we should be confortable just adding the major and letting `Cargo` take the latest version (which may include performance improvements or bug fixes)